subgraphs/carbon/deployments.ts

```diff
@@ -0,0 +1,64 @@
+interface Contract {
+    address: string;
+    startBlock: number;
+}
+
+interface AmmConfig {
+    Controller: Contract;
+    Voucher: Contract;
+}
+
+interface ChainConfig {
+    nativeToken: {
+        address: string;
+        symbol: string;
+        name: string;
+        decimals: number;
+    };
+    amms: { [amm: string]: AmmConfig };
+}
+
+export const DEPLOYMENTS: {
+    [network: string]: ChainConfig;
+} = {
+    sepolia: {
+        nativeToken: {
+            address: "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
+            symbol: "ETH",
+            name: "Ether",
+            decimals: 18,
+        },
+        amms: {
+            carbon: {
+                Controller: {
+                    address: "0x5bCA3389786385a35bca14C2D0582adC6cb2482e",
+                    startBlock: 6969415,
+                },
+                Voucher: {
+                    address: "0x59f21012B2E9BA67ce6a7605E74F945D0D4C84EA",
+                    startBlock: 6969374,
+                },
+            },
+        },
+    },
+    "sei-mainnet": {
+        nativeToken: {
+            address: "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
+            symbol: "SEI",
+            name: "Sei",
+            decimals: 18,
+        },
+        amms: {
+            carbon: {
+                Controller: {
+                    address: "0xe4816658ad10bF215053C533cceAe3f59e1f1087",
```

Comment:
Are these addresses correct? I can't find them on the SEI scan explorer.

Reply:
You can check them out on seitrace.com